src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -55,13 +55,13 @@ const SelectPanel = props => {
   React.useEffect(() => setInternalOpen(props.open), [props.open])
 
   const onInternalClose = () => {
-    if (props.open === 'undefined') setInternalOpen(false)
+    if (props.open === undefined) setInternalOpen(false)
     if (typeof props.onCancel === 'function') props.onCancel()
   }
   // @ts-ignore todo
   const onInternalSubmit = event => {
     event.preventDefault()
-    if (props.open === 'undefined') setInternalOpen(false)
+    if (props.open === undefined) setInternalOpen(false)
```

Comment:
I think you fixed it in the other PR, no? https://github.com/primer/react/pull/3770

Reply:
This is that PR :)

Reply:
>This is that PR :)
🙈